exercises/high-scores/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[dependencies]
+
+[package]
+authors = ["Oleksii Filonenko <brightone@protonmail.com>"]
```

Comment:
Please remove this line; policy since https://github.com/exercism/rust/issues/524 has been to leave it out.

Reply:
Okay, sure. `exercise generate` put it there for me. Maybe it should get removed automatically?